content/docs/guides/components/hyperparameter.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "Hyperparameter Tuning (Katib)"
+description = "Hyperparameter Tuning (Katib)"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+
+## Deploying Katib
+
+[Katib](https://github.com/kubeflow/katib) is a hyperparameter tuning framework, inspired by
+[Google Vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf).
+
+To deploy katib,
+```shell
+ks pkg install kubeflow/katib@master
+ks generate katib katib
+ks apply YOUR_ENV -c katib
```

Comment:
nit: YOUR_ENV -> ${ENV}
This way its a proper bash command

Reply:
Sure. I simply copy pasted what is there at: https://github.com/kubeflow/kubeflow/blob/master/kubeflow/katib/README.md.